portfolio/src/main/webapp/style.css

```diff
@@ -1,78 +1,198 @@
+  text-align: center;
+  left: 7.5%;
+  width: 85%;
+  height: 95vh;
+  background-color: rgba(0, 128, 128, .6);
+}
+
+#icons {
+  padding-top: .5%;
+}
+
+#headshot {
+  width: 100%;
+  height: 100%;
+}
+
+.front-tile-logo {
+  width: 40%;
+  height: 40%;
 }
 
-/*Should find some generalizable way to autofit text in this box*/
+/*We should find some generalizable way to autofit text in this box.*/
+
 .tile {
-  float:left;
-  width: 31%;
-  height: 31%;
-  padding:1%;
-  background-clip:content-box; 
-  /*this handy line makes sure that padding is not 
-  also colored by the background color*/ 
-  background-color:yellow; 
+  /*Style was changed to have .33% at end so elements are perfectly centered
+  with that value of padding.*/
+  float: left;
+  width: 31.33%;
+  height: 31.33%;
+  padding: 1%;
+  vertical-align: middle;
+  background-clip: content-box;
+  /*This handy line makes sure that padding is not also colored by the
+  background color.*/
+  background-color: rgba(255, 255, 0, .65);
+}
 
+.portrait-tile-back {
+  overflow: auto;
+  width: 100%;
+  height: 100%
+}
+
+.tile-back img, .tile-front img, .portrait-tile-back img {
+  width: 100%;
+  height: 100%;
+  /*Uncomment line if don't want to distort image resolution.*/
+  /*object-fit:cover;*/
+}
+
+.title {
+  font-size: 4em;
+  font-family: fantasty;
+}
+
+#hammer-tile-logo {
+  width: 64%;
+  height: 40%;
+  padding-top: 4%;
+  height: 79%;
+}
+
+#juggling-tile-logo {
+  width: 90%;
+  height: 73%;
+  padding-top: 4%;
 }
-#Headshot{
-  position:relative;
-  width:12.5%;
+
+/*This styles the iframe body for including the large background content.*/
+
+html:not(root) body {
+  overflow-x: hidden;
+  overflow-y: auto;
+  width: 100%;
+  height: 100%;
 }
-.tileInner {
+
+ul {
+  list-style: none;
+}
+
+.portrait-tile {
+  width: 13vw;
+  height: 15vw;
+  margin-left: 36vw;
+}
+
+.portrait-tile-front img {
+  width: 100%;
+  height: 100%;
+}
+
+.portrait-tile-back div {
+  overflow: auto;
+  width: 100%;
+  height: 100%
+}
+
+.tile-inner, .portrait-tile-inner {
   position: relative;
   width: 100%;
   height: 100%;
   text-align: center;
   transition: transform 0.8s;
   transform-style: preserve-3d;
-  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
 }
 
-.tile:hover .tileInner{
+.tile-inner {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+}
+
+.portrait-tile-inner {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+}
+
+.tile:hover .tile-inner {
   transform: rotateX(180deg);
 }
 
-.tileFront, .tileBack {
+.portrait-tile:hover .portrait-tile-inner {
+  transform: rotateY(180deg);
+}
+
+.tile-front, .tile-back, .portrait-tile-back, .portrait-tile-front {
   position: absolute;
   width: 100%;
   height: 100%;
   -webkit-backface-visibility: hidden;
   backface-visibility: hidden;
-  font-size: .8vw; 
+  font-size: .8vw;
 }
 
-.tileFront {
-  /*background-color: #bbb;*/
-  /*color: black;*/
+.tile-front {
+  /*This is left empty in anticipation of further styling.*/
 }
```

Comment:
I'd recommend against placeholders like this. Generally the code is simpler if you just add things when you need them. (Though, of course it's good to _think_ about how things will look when you add features in the future.)